docker-compose.yml

```diff
@@ -138,6 +138,7 @@ services:
       - COIN=ETH
       - ETHEREUM_JSONRPC_VARIANT=geth
       - ETHEREUM_JSONRPC_HTTP_URL=http://zkevm-explorer-json-rpc:8124
+      # - ETHEREUM_JSONRPC_HTTP_URL=http://zkevm-explorer-json-rpc:8545
```

Comment:
this commented line allows the explorer to use the L1 container as the data source.